study.md

```diff
@@ -12,6 +12,8 @@ material. Instead, digest what you've read and repeat it in your own voice.
 
 -   [What the heck is the event loop anyway?](https://www.youtube.com/watch?v=8aGhZQkoFbQ)
 -   [The Restaurant](https://www.codeschool.com/blog/2014/10/30/understanding-node-js/)
+-   [Node Hero](https://blog.risingstack.com/node-hero-async-programming-in-node-js/)
```

Comment:
I think maybe add a note that we are not going to be programming in Node RIGHT NOW, and this is relevant to all async programming?